### DISCUSSION TITLE
Icon Request: Auction / Hammer

### DISCUSSION THREAD

**Discussion by d-m-s** [vmware/clarity#5831] [Ideas] [answered]
## Summary

Icon to represent auction / auction-hammer.
- What is the icon?
   auction-hammer 
   
- What does the requested icon signify?
   a hammer, like it is used during an auction

## Use case
   - show that an item is related to an auction; 
   - start an action related to an auction.

## Examples
<img width="51" alt="2021-04-12_14h50_33" src="https://user-images.githubusercontent.com/311619/114411987-ff9b9a80-9bac-11eb-916c-2104c99434da.png">

**Comment by sis0k0** [comment]
Hi, @d-m-s! Thank you for the suggestion! We'll discuss it with the team and get back to you.

**Reply by d-m-s** [reply]
hi @sis0k0, do you have any news on the icon? thx!

**Reply by bbogdanov** [reply]
@lil-kim

**Reply by lil-kim** [reply]
Hi @d-m-s ,
I will have this icon done tomorrow.

**Reply by d-m-s** [reply]
that's great news, thx!

**Comment by hippee-lee** [comment, marked as answer]
@lil-kim - this is an external icon request. Do you think it is something we should add?

**Reply by lil-kim** [reply]
Sure.

**Comment by lil-kim** [comment]
This icon is completed and ready for implementation in Epic #5883
